public/pages/workflow_detail/workflow_detail.tsx

```diff
@@ -109,8 +109,7 @@ export function WorkflowDetail(props: WorkflowDetailProps) {
   // - fetch available models as their IDs may be used when building flows
   useEffect(() => {
     if (!isNewWorkflow) {
-      // TODO: can optimize to only fetch a single workflow
-      dispatch(searchWorkflows(FETCH_ALL_QUERY_BODY));
+      dispatch(getWorkflow(workflowId));
```

Comment:
Right, it will. This plugin is still in early stages, so the first step is to get all of the API calls working the basic way. Once there is finalized design on the initial pages and user flows, MDS support will be added altogether.

Reply:
are we releasing the basic way like this for 2.14?

Reply:
There's no timelines yet since there is still no final / initial proposed UX and user flows.

Reply:
I think both Sean and Minal are under the impression that flow framework frontend will be MDS native as this is a new plugin. We might want to correct this impression.

Reply:
Let's keep this open source and tag github users here. It will be MDS native, there is no wrong impression.